code/src/components/Text.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './text.css'
+
+export const Text = (props) => {
+    return(
+        <>
+        <h2 tabIndex={'0'}>What did you think about this awesome survey:</h2>
+        <div className="text-input">
+            <input tabIndex={'0'} type="text" onChange={(event) => props.setUserText(event.target.value)}/>
+        </div>
+        </>
+    )
+}
+/* Function for next question button:
+const nextQuestion = () => setQuestion(question + 1) //to get next question in line (add +1 later) const previousQuestion = () => setQuestion(question - 1) */
```

Comment:
I think you can move this comment to Survey.js where you have declared these functions if you feel that you want to keep the comment, if not I think you can remove it.